README.md

```diff
@@ -9,23 +9,159 @@ An engineer working for a [CSE](./CSE.md) project...
 
 This is the playbook. You are invited to contribute via standard git pull requests to GitHub.
 
-## The Basics
-
-* [Source Control](./Engineering/SourceControl.md)
-* [Unit Testing](./Engineering/UnitTesting.md)
-* [Continuous Integration / Continuous Delivery](./Engineering/CICD.md)
-* [Code Reviews](./Engineering/CodeReviews.md)
-* [Retrospectives](./Engineering/Retrospectives.md)
-* [Engineering Practices And Principles](./Engineering/PracticiesAndPrinciples.md)
-
-## Seeking Contributions for
-
-- Component Tests
-- DevOps
-    - [Component Versioning](./Engineering/ComponentVersioning.md)
-    - [Logging and Monitoring](./Engineering/DevOpsLogging.md)
-    - Performance
-    - Telemetry
-- Security
-    - Threat Model
-- Regulatory Compliance
+## Outline
+
+The following outlines each section and what content would be included within each section.
+
+### Goals
+
+The proposed outline is intended to accomplish the following goals
+
+1. Author as idea-to-delivery value stream narrative from an engineering perspective.
+2. Organize for quick reference and discoverability.
+3. Extensible heirarchy to allow teams to share deep subject matter expertise.
+
+### Working Heirarchy
+
+The working heirarchy is rooted in CSE's one week sprint cycle. Each section within then allows for deep dive information, recipes, and use cases.
```

Comment:
"The working heirarchy is rooted in CSE's one week sprint cycle, though the ordering and general guidance should still easily apply to sprint cycles of any reasonable length."

Reply:
added your clarification